content/docs/command-reference/exp/init.md

```diff
@@ -19,43 +18,57 @@ usage: dvc exp init [-h] [-q | -v] [--run] [--interactive] [-f]
 
 ## Description
 
-`dvc exp init` helps you quickly get started with experiments. It reduces
-boilerplate for initializing [pipeline](/doc/command-reference/dag) stages in a
-`dvc.yaml` file by assuming defaults about the location of your data,
-[parameters](/doc/command-reference/params), source code, models,
-[metrics](/doc/command-reference/metrics) and
-[plots](/doc/command-reference/plots), which can be customized through config.
+`dvc exp init` helps you get started with DVC Experiments quickly. It reduces
+boilerplate DVC procedures by creating a `dvc.yaml` file that assumes standard
+locations of your input data, <abbr>parameters</abbr>, source code, models,
+<abbr>metrics</abbr> and [plots](/doc/command-reference/plots). These locations
+can be customized through the [options](#options) below or via
+[configuration](/doc/command-reference/config#exp).
 
-It also offers guided `--interactive` mode for creating a stage to be
-[`exp run`](/doc/command-reference/exp/run) later. `dvc exp init` supports
-creating different types of stages, eg: `dl` if you are doing deep learning,
-which uses [dvclive](/doc/dvclive) to monitor and checkpoint progress during
-training of machine learning models.
+Assumed repository structure:
 
-This command is intended to be a quick way to start running experiments. To
-create more complex stages and pipelines, use `dvc stage add`.
+```
+├── data/
+├── metrics.json
+├── models/
+├── params.yaml  # required by default
+├── plots/
+└── src/
+```
+
+> Note that by default `dvc exp init` expects at least a `params.yaml` file
+> present. It can be avoided via the `--params`, `-i`, or `--explicit` options.
```

Comment:
I like the explanation under `--params`, and I'm not sure this explains enough about how to solve the issue. Maybe something like:

Note that by default `dvc exp init` expects at least a `params.yaml` file present. This is because DVC reads that file to find parameters to include in the stage. Modify the parameters file path via `--params`, omit all default paths via `--explicit`, or use `-i` to configure via prompts.

Reply:
Updated in 600e83ca.